.travis.yml

```diff
@@ -1,8 +1,14 @@
 rvm:
-  - 1.9.2
-  - 1.9.3
   - 2.0
-  - 2.1.0
+  - 2.1.7
+  - 2.2.3
+addons:
+  apt:
+    packages:
+      - postgresql-server-dev-9.3
```

Comment:
As travis migrated to container based environment, postgresql-server-dev package is needed to be installed to install do_postgres.